docs/shared/tools-workspace-builders.md

```diff
@@ -51,18 +51,18 @@ In our `impl.ts` file, we're creating an `Options` interface that matches the js
 The `impl.ts` contains the actual code for your executor. Your executor's implementation must export a function that takes an options object and returns a `Promise<{ success: boolean }>`.
 
 ```typescript
-import { ExecutorContext } from '@nrwl/devkit';
+import type { ExecutorContext } from '@nrwl/devkit';
 import { exec } from 'child_process';
 import { promisify } from 'util';
 
-export interface EchoExecutorOptions {
+export type interface EchoExecutorOptions {
```

Comment:
What's `type interface`?

Reply:
there's `export` before type,
EchoExecutorOptions is a type and not a value that is used during runtime so we can use `export type` here.

Reply:
But why is it needed? It's an interface, it's already just a type

Reply:
True 🤦,
I feel stupid

Reply:
All good, it was just weird for me to see, I'm not sure it's even valid syntax tbh 😅

Reply:
It's not, just checked

Reply:
Good catch!